Kitodo-DataManagement/src/main/java/org/kitodo/data/database/persistence/ProcessDAO.java

```diff
@@ -105,6 +105,24 @@ public void update(Process process) {
         return getByQuery("FROM Process WHERE template = 1 AND inChoiceListShown = 1 ORDER BY title ASC");
     }
 
+    /**
+     * Get all processes.
+     *
+     * @return list of all processes as Process objects
+     */
+    public List<Process> getProcesses() {
+        return getByQuery("FROM Process WHERE template = 0 ORDER BY title ASC");
+    }
+
+    /**
+     * Get all processes.
```

Comment:
Please adjust the JavaDoc comment as this method is only retrieving `active` processes.